### ISSUE TITLE
Annotation not performed with hg19

### ISSUE THREAD

**Issue by paumarc** [nf-core/sarek#953]
### Description of the bug

VEP does not work with hg19 (at least,  other genomes has not been tested), neither when all pipeline is executed, nor when the annotation step is launched

### Command used and terminal output

```console
nextflow run nf-core/sarek -profile singularity --input results_Lpalomo_Wes_no_intervals/csv/variantcalled.csv  --genome hg19 -r 3.1.2 -c sarek.conf --outdir results_Lpalomo_Wes_no_intervals/ --tools vep -resume  --step annotate

and the sample sheet

patient,sample,vcf
WES_200209,WES_200209,results_Lpalomo_Wes_no_intervals/variant_calling/strelka/WES_200209/WES_200209.strelka.variants.vcf.gz
WES_200394,WES_200394,results_Lpalomo_Wes_no_intervals/variant_calling/strelka/WES_200394/WES_200394.strelka.variants.vcf.gz
WES_201085,WES_201085,results_Lpalomo_Wes_no_intervals/variant_calling/strelka/WES_201085/WES_201085.strelka.variants.vcf.gz
WES_201184,WES_201184,results_Lpalomo_Wes_no_intervals/variant_calling/strelka/WES_201184/WES_201184.strelka.variants.vcf.gz
WES_210490,WES_210490,results_Lpalomo_Wes_no_intervals/variant_calling/strelka/WES_210490/WES_210490.strelka.variants.vcf.gz
results_Lpalomo_Wes_no_intervals/csv/variantcalled.csv (END)
```

### Relevant files

[nextflow.log](https://github.com/nf-core/sarek/files/10821994/nextflow.log)

### System information

singularity --version
singularity-ce version 3ea83f6-dirty

Linux bioinf.vhio.org 5.4.0-91-generic #102-Ubuntu SMP Fri Nov 5 16:31:28 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

**Comment by maxulysse** [comment]
Hi @paumarc so I tried with:

```
nextflow run nf-core/sarek -r dev -latest -profile test,docker --outdir results --step annotate --tools vep --genome hg19 --vep_cache_version 106 --vep_genome GRCh37 --vep_species homo_sapiens --vep_version 106.1 --input results/csv/variantcalled.csv
```

And I do see a VEP process happening.

**Comment by paumarc** [comment]
Hi @maxulysse 

It gave me many errors after installing last nextflow version locally at the directory

WARN: There's no process matching config selector: .*:FREEC_SOMATIC -- Did you mean: FREEC_SOMATIC?
WARN: There's no process matching config selector: .*:FILTERVARIANTTRANCHES -- Did you mean: FILTERVARIANTTRANCHES?
WARN: There's no process matching config selector: NFCORE_SAREK:SAREK:(BAM_MARKDUPLICATES|BAM_MARKDUPLICATES_SPARK):CRAM_QC_MOSDEPTH_SAMTOOLS:SAMTOOLS_STATS
WARN: There's no process matching config selector: NFCORE_SAREK:SAREK:CRAM_QC_NO_MD:SAMTOOLS_STATS
WARN: There's no process matching config selector: NFCORE_SAREK:SAREK:CRAM_TO_BAM
No such file: /mnt/bioinfnas/bioinformatics/projects/20221117_LPalomo_cfDNA_limfoma/WES/results_Lpalomo_Wes_no_intervals/variant_calling/strelka/WES_200209/WES_200209.strelka.variants.vcf.gz

 -- Check script '/home/paumunoz@vhio.org/.nextflow/assets/nf-core/sarek/./workflows/sarek.nf' at line: 1316 or see '.nextflow.log' file for more details
[nextflow.log](https://github.com/nf-core/sarek/files/10852285/nextflow.log)

**Comment by maxulysse** [comment]
ok, the WARNING are not important, but what worries me is the `No such file`.
Can you try:
```
nextflow run nf-core/sarek -r dev -latest -profile singularity --outdir results/ --step annotate --tools vep --genome hg19 --vep_cache_version 106 --vep_genome GRCh37 --vep_species homo_sapiens --vep_version 106.1 --input --input results_Lpalomo_Wes_no_intervals/csv/variantcalled.csv
```
in a way that input points to your own `variantcalled.csv`

**Comment by maxulysse** [comment]
using params managed to fix the missing keys, so merging #954 is a go
